src/mlpack/core/tree/hollow_ball_bound_impl.hpp

```diff
@@ -0,0 +1,384 @@
+/**
+ * @file hollow_ball_bound_impl.hpp
+ *
+ * Bounds that are useful for binary space partitioning trees.
+ * Implementation of HollowBallBound ball bound metric policy class.
+ *
+ * @experimental
```

Comment:
The @experimental mark doesn't need to be here... I think hrectbound_impl.hpp has it too, it's probably not needed there either.